src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.lucene.query.recipe.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "Blog.Lucene",
-  "displayName": "Blog - Lucene",
-  "description": "Runs Lucene setup recipe and replaces RecentBlogPosts SQL query with Lucene.",
+  "name": "Blog.Lucene.Query",
```

Comment:
After running the recipe the query is empty:
![image](https://user-images.githubusercontent.com/1976647/162324084-eb4cb4bf-3924-4899-bd96-876081233424.png)

Reply:
You need to reset the Search Lucene index because the content item was created before the Lucene indices.
![image](https://user-images.githubusercontent.com/79921246/162415249-b71799fd-f295-4005-a6db-11df1cd08de7.png)

Reply:
This can be done from the recipe, so it's not a necessary manual step, no?

Reply:
I look through the code, and the Lucene recipe step creates the index but doesn't process the old content items. I can modify the [LuceneIndexStep](https://github.com/OrchardCMS/OrchardCore/blob/339d74b251b53393dd510f8ad3027aa17d78500d/src/OrchardCore.Modules/OrchardCore.Lucene/Recipes/LuceneIndexStep.cs#L46) and make it possible, but I am not sure if there is any reason we haven't done it before.